expense-notifications-functions/src/index.ts

```
@@ -0,0 +1,476 @@
+async function getUserDisplayName(userId: string) {
+  if (!userId) {
+    logger.error('Invalid userId:', userId);
+    return 'Unknown';
+  }
+
+  try {
+    const userDoc = await db.collection('users').doc(userId).get();
+
+    if (userDoc.exists) {
+      const userData = userDoc.data();
+      return userData && userData.first_name ? userData.first_name : 'Unknown';
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)